backend/apps/accounts/migrations/0021_merge_20260714_1801.py

```
@@ -0,0 +1,14 @@
+# Generated by Django 5.0.14 on 2026-07-14 12:31
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('accounts', '0015_case_insensitive_username_unique'),
+        ('accounts', '0020_alter_userprofile_jwt_token_version_and_more'),
+    ]
```

Ruff (0.15.21)

[warning] Mutable default value for class attribute

(RUF012)